Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -1487,17 +1487,15 @@ HTML::EnvironmentSettingsObject& Document::relevant_settings_object() const
 }
 
 // https://dom.spec.whatwg.org/#dom-document-createelement
-WebIDL::ExceptionOr<JS::NonnullGCPtr<Element>> Document::create_element(String const& a_local_name, Variant<String, ElementCreationOptions> const& options)
+WebIDL::ExceptionOr<JS::NonnullGCPtr<Element>> Document::create_element(String local_name, Variant<String, ElementCreationOptions> const& options)
```

Comment:
If we're not `move`ing the string anywhere, let's keep it a `const&`

Reply:
Did you mean to totally revert Document.cpp?

Reply:
Yeah, I just dopped this one. The copy was because we need to make a copy in order to `to_lowercase` it. I suppose I could have reordered the steps as well to keep it as a `const&` though. For now, I just dropped this commit though (I had accidentally dropped the other change in Document though)

Reply:
Oh I see. For the record, that reason ("I'm mutating it!") would've been a fine response to my original comment :sweat_smile: